### PR TITLE
feat(prettier): add

### PR DESCRIPTION
**Pull request by JAdshead**
Create a Prettier friendly eslint config

### CODE REVIEW

File: prettier.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
```
Comment (JamesSingleton) [inline prettier.js:2]: Should this be 2020?

File: prettier.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+module.exports = {
+  extends: [
+    './index',
+    'prettier',
```
Comment (JamesSingleton) [inline prettier.js:18]: Prettier should be a dep right?

File: prettier.js (hunk 1 above)
Comment (JAdshead) [reply prettier.js:2]:
```suggestion
 * Copyright (c) 2020 American Express Travel Related Services Company, Inc.
```

File: prettier.js (hunk 2 above)
Comment (JAdshead) [reply prettier.js:18]: only if we want to use prettier in this lib

Comment (JamesSingleton) [reply prettier.js:18]: So then it should be a `peerDependency` then?

**Comment by enuchi** [comment]
@JAdshead looks like the capital "C" Copyright failed the commit message check.

File: prettier/test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+module.exports = {
+  extends: [
+    './index',
```
Comment (enuchi) [inline prettier/test.js:17]: shouldn't this config for tests extend prettier?

Comment (JAdshead) [reply prettier/test.js:17]: yeah it extends from the index file inside `prettier`

Comment (enuchi) [reply prettier/test.js:17]: ah yes 👍

**Pull request merged**